api/routes/expenseRoutes.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const expenseController = require('../controllers/expenseController');
+let expenseRouter = express.Router();
+
+// EndPoint: /api/expense/
+
+expenseRouter.route('/').get(expenseController.getExpenses);
+
+expenseRouter.route('/').post(expenseController.insertExpense);
```

Comment:
you can actually chain on routes using the same path like
```
expenseRouter.route('/')
  .get(expenseController.getExpenses)
  .post(expenseController.insertExpense);
```

Reply:
I'll add it to the code.